modules/nw-sriov-configuring-device.adoc

```diff
@@ -87,6 +88,13 @@ A device can be used in either mode.
 ====
 endif::virt-sriov[]
 
+. (Optional) Label the SR-IOV cluster nodes if they are not already labeled:
++
+[source,terminal]
+----
+$ oc label node SriovNetworkNodePolicy.spec.nodeSelector
```

Comment:
sorry, I didnt make it clear.
The `SriovNetworkNodePolicy.spec.nodeSelector` is not executable in the above cmd. I use it to refer to the nodeSelector field in SriovNetworkNodePolicy, would you mind make this as a normal context source (or any other source you think is appropriate), instead of terminal source?